lib/SIL/Verifier/SILVerifier.cpp

```diff
@@ -4748,6 +4748,70 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
                     "Type of witness instruction does not match actual type of "
                     "witnessed function");
   }
+  
+  void checkGetAsyncContinuationInstBase(GetAsyncContinuationInstBase *GACI) {
+    auto resultTy = GACI->getType();
+    auto &C = resultTy.getASTContext();
+    auto resultDecl = resultTy.castTo<BoundGenericType>()->getDecl();
```

Comment:
This will crash if it isn't a bound generic type; I think there might be a helper to make this easier to check safely.